gitlab/docs/runner/histogram.gitlab_runner_autoscaling_machine_creation_duration_seconds_sum.md

```diff
@@ -0,0 +1,8 @@
+---
+title: gitlab_runner_autoscaling_machine_creation_duration_seconds_sum
+brief: Histogram of machine creation time.
+metric_type: histogram
```

Comment:
General comment - our only metric types are gauge, counter, and cumulative counter. I see histogram, summary, and maybe others